app/data/europe_sconly_start.json

```diff
@@ -1,8 +1,9 @@
 ﻿{
     "date": {
-        "year": "1900",
+        "year": 0,
         "season": "Winter",
         "phase": "Adjustment"
+        "base": 1900
```

Comment:
instead of defining a "base" (baseYear, if anything), I propose we keep to `year: 1900`, and define a `startDate` or `startYear` in the defs instead.